spec/support/fetch_secrets_helper.rb

```diff
@@ -18,7 +18,7 @@
 
   let(:mocked_secret) do
     double('Secret').tap do |secret|
-      allow(secret).to receive(:value).and_return("mocked-secret")
+      allow(secret).to receive(:value).and_return(+"mocked-secret") # unfreezing the secret for authn-azure tests
```

Comment:
1. Put comment above so line doesn't exceed 80 chars.
2. Can you explain more how this will be used, because my first instinct is that there should never be a use-case where you need to mutate strings.

Reply:
1. Done.
2. Please see here - https://github.com/cyberark/conjur/pull/1406